[BE][cutlass backend] Reduce severity of log message for no cutlass config found

henrylhtsang opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #160148

This is not really a problem. Sometimes we cannot find a cutlass config due to shape, e.g. when k is odd. 


cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @ipiszy @chenyang78 @kadeng @muchulee8 @amjames @chauhang @aakhundov @coconutruben

henrylhtsang committed:
[BE][cutlass backend] Reduce severity of log message for no cutlass config found

[ghstack-poisoned]

Skylion007 commented on torch/_inductor/codegen/cuda/gemm_template.py:
```diff
@@ -598,7 +598,7 @@ def _add_cutlass_gemm_choices(
             input_strides = [node.get_stride() for node in input_nodes]
             output_layout = layout
             warning_msg = f"No suitable Cutlass GEMM configs found, fallbacks used ( {len(ops)=}, {output_layout=}, {input_layouts=}, {input_strides=} )"  # noqa: B950
```
Nit: this warning message is constructed whether it's needed to for logging. Use Loggers builtin string interpolation with lazy %s formatting. We also have ruff statically verify it.

Skylion007 reviewed (approved):
Logging perf nit

henrylhtsang committed:
Update on "[BE][cutlass backend] Reduce severity of log message for no cutlass config found"


This is not really a problem. Sometimes we cannot find a cutlass config due to shape, e.g. when k is odd. 


cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy chenyang78 kadeng muchulee8 amjames chauhang aakhundov coconutruben

[ghstack-poisoned]
